components/NewsItem.vue

```diff
@@ -2,10 +2,11 @@
 	<!-- This compenent is used by data from pages api call as well as post api call -->
 	<article>
 		<figure class="preview-media">
-			<img :src="picture" :alt="altTag" class="news-item-img">
+			<img v-if="picture" :src="picture" :alt="altTag">
+			<WeCountLogo v-else class="news-item-img" />
```

Comment:
Shall the same class `news-item-img` be applied to the <img> tag at line 5 as well?

Reply:
By applying the same class, the css can be cleaned up by removing the `figure img` at line 8 in _news-and-views.scss.

Reply:
True! Okay, I'll implement this